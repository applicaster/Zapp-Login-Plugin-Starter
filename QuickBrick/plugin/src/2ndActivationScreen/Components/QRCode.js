import * as React from 'react';
import { Image } from 'react-native';

export default function QRButton({ url = '' }) {
  return (
    <Image
      style={{ width: 340, height: 340 }}
      resizeMode="contain"
      source={{ uri: url }}
    />
  );
}
