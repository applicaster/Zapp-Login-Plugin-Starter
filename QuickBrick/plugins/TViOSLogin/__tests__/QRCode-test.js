import React from 'react';
import renderer from 'react-test-renderer';
import QRCode from '../src/2ndActivationScreen/Components/QRCode';


it('QRCode renders correctly', () => {
  const tree = renderer
    .create(<QRCode />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
