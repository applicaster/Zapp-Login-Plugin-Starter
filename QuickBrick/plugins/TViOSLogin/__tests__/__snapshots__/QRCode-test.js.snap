// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QRCode renders correctly 1`] = `
<Image
  resizeMode="contain"
  source={
    Object {
      "uri": "",
    }
  }
  style={
    Object {
      "height": 340,
      "width": 340,
    }
  }
/>
`;
